bpf/process/types/basic.h

```diff
@@ -1578,6 +1578,8 @@ FUNC_INLINE int match_binaries(__u32 selidx)
 
 		switch (selector_options->op) {
 		case op_filter_in:
+			if (current->bin.mb_bitset & (1UL << selector_options->mbset_id))
+				return 1;
```

Comment:
A comment here would be welcomed to say that you short circuit matching since it's a child of a followed process. And I don't if we can write it but an explicit `[[fallthrough]]` would be amazing.

Reply:
It does not look like `[[fallthrough]]` is supported.

Added:
```
                case op_filter_in:
+                       /* Check if we match the selector's bit in ->mb_bitset, which means that the
+                        * process matches a matchBinaries section with a followChidren:true
+                        * attribute either because the binary matches or because the binary of a
+                        * parent matched.
+                        */
                        if (current->bin.mb_bitset & (1UL << selector_options->mbset_id))
                                return 1;
+                       /* fallthrough */
```